cmd/influxd/server_integration_test.go

```diff
@@ -1461,7 +1461,7 @@ func TestSingleServerDiags(t *testing.T) {
 	config := main.NewConfig()
 	config.Monitoring.Enabled = true
 	config.Monitoring.WriteInterval = main.Duration(100 * time.Millisecond)
-	nodes := createCombinedNodeCluster(t, testName, dir, 1, config)
+	nodes := createCombinedNodeCluster(t, testName, dir, 3, config)
```

Comment:
This change ensures diags and stats self-monitoring works for both single and multiple node cluster.